worli.sh

```diff
@@ -1,266 +1,259 @@
 #!/bin/bash
 if [ "$EUID" -ne 0 ]
-  then echo "this script needs to be ran as root. if your a noob thats 'sudo ./worli.sh'"
-  exit
+    then echo "This script needs to be run as root. If you're a noob, that's 'sudo ./worli.sh'"
+    exit 1
 fi
-echo  "WORli, made by JoJo Autoboy#1111"
-echo  "heavily based off of Mario's wor linux guide: https://www.worproject.ml/guides/how-to-install/from-other-os"
+echo "WoRli, made by JoJo Autoboy#1931"
+echo "Heavily based off of Mario's WoR Linux guide: https://worproject.ml/guides/how-to-install/from-other-os"
 
 mkdir -p /tmp/isomount
 chmod 777 /tmp/isomount
 
 echo " "
-echo -e "are you installing to a Raspberry Pi 4 or a Raspberry Pi 3, CM3 or 2?"
-read -r -p "[4/3]: " input
+echo -e "Are you installing to a Raspberry Pi 4, 3 (including CM3), or 2?"
+read -r -p "[4/3/2]: " input
 case $input in
     [4])
- export PI="4"
- ;;
+    export PI="4"
+    ;;
     [3])
- export PI="3"
-       ;;
+    export PI="3"
+    ;;
+    [cC][mM][3])
+    export PI="3"
+    ;;
+    [2])
+    export PI="3"
+    ;;
     *)
- echo "Invalid input..."
- exit 1
- ;;
-esac 
+    echo "Unsupported model"
+    exit 1
+    ;;
+esac
 
 echo " "
-echo -e "do you want the tool to download the uefi, drivers and pe-installer or use your own files?"
+echo -e "Do you want the tool to download the UEFI, drivers, and PE-installer automatically? Say 'N' to use your own files"
```

Comment:
, and 
need i say more?

Reply:
Go learn English

Reply:
i'm not arguing over a comma but i remember that not being a thing you can do

Reply:
wait what